src/runtime/EncodedValue.h

```diff
@@ -235,21 +235,25 @@ class EncodedValue {
         return HAS_SMI_TAG(m_data.payload);
     }
 
-    uint32_t asInt32()
+    bool isUInt32()
+    {
+        return isInt32() && asInt32() >= 0;
```

Comment:
Can't the ```UInt32``` mentioned here actually use all 32 bits? If so, I think users using it may be confused.
How about adding comments for a more detailed description?

Reply:
Thanks for the review.
I add some comments for it.